lisa/messages.py

```diff
@@ -38,6 +38,19 @@ class TestRunMessage(MessageBase):
 @dataclass
 class PerfMessage(MessageBase):
     type: str = "Performance"
+    test_case_name: str = ""
+    platform: str = ""
+    location: str = ""
+    host_version: str = ""
+    guest_os_type: str = "Linux"
+    distro_version: str = ""
+    vmsize: str = ""
+    kernel_version: str = ""
+    lis_version: str = ""
+    ip_version: str = "IPv4"
```

Comment:
Add Enum class for ip_version, protocol_type like below

```python
class NetworkProtocol(str, Enum):
    IPv4 = "IPv4"
    IPv6 = "IPv6"

class TransportProtocol(str, Enum):
    Tcp = "TCP"
    Udp = "UDP"
```

Reply:
update